src/collective/fingerpointing/upgrades/v2/__init__.py

```diff
@@ -1,19 +1,19 @@
 # -*- coding: utf-8 -*-
 from collective.fingerpointing.config import PROJECTNAME
-from collective.fingerpointing.logger import logger
+from collective.fingerpointing.logger import log_info
 
 
 def update_configlet(setup_tool):
     """Update control panel configlet."""
     profile = 'profile-{0}:default'.format(PROJECTNAME)
     setup_tool.runImportStepFromProfile(profile, 'controlpanel')
-    logger.info('Control panel configlet updated.')
+    log_info.logger.info('Control panel configlet updated.')
```

Comment:
we don't want this information logged into the audit log.

Reply:
Its the same as before. A logger with the same name is a singleton.  If we don’t want it, I change it to use the commonlogger.